openlibrary/plugins/worksearch/code.py

```diff
@@ -309,11 +309,9 @@ def parse_json(raw_file):
         return None
     return json_result
 
-def run_solr_query(param = {}, rows=100, page=1, sort=None, spellcheck_count=None, offset=None, fields=None):
-    # called by do_search
-    if spellcheck_count == None:
-        spellcheck_count = default_spellcheck_count
 
+def run_solr_query(param = {}, rows=100, page=1, sort=None, spellcheck_count=None,
```

Comment:
See #3613

Reply:
I think that's just a bug; thanks for opening a new issue!